src/lib.rs

```diff
@@ -202,6 +209,7 @@ impl IronOxide {
                 BYTES_BEFORE_RESEEDING,
                 EntropyRng::new(),
             )),
+            runtime: rt,
```

Comment:
Nitpick: Is there a reason not to name the above variable 'runtime' to make this shorter?